manim/utils/cfgwriter.py

```diff
@@ -34,10 +36,36 @@ def check_valid_style(style):
         return False
 
 
+def replace_keys(default):
+    """Replaces _ to . and viceversa in a dictionary for rich
+    Parameters
+    ----------
+    default : :class:`dict`
+        The dictionary to check and replace
+    Returns
+    -------
+    :class:`dict`
+        The dictionary which is modified by replcaing _ with . and viceversa
+    """
+    for key in default:
+        if "_" in key:
+            temp = default[key]
+            del default[key]
+            key = key.replace("_", ".")
+            default[key] = temp
+        else:
+            temp = default[key]
+            del default[key]
+            key = key.replace(".", "_")
+            default[key] = temp
+    return default
+
+
 def main():
-    successfully_read_files = _run_config()[-1]
+    config_items = _run_config()
+    successfully_read_files = config_items[-1]
     console = Console()
-    config = configparser.ConfigParser()
+    config = config_items[1]
```

Comment:
Nitpick: it's more pythonic to do `_, config_parser, _, successfully_read_files = _run_config(). This is preferable since it explicitly says which return values of `_run_config` are being used and which are being discarded.